internal/handler/tools/handler.go

```diff
@@ -312,10 +338,10 @@ func (h *Handler) RegisterServiceHandlers(s *server.MCPServer) {
 	})
 
 	getOpsTool := mcp.NewTool("get_service_top_operations",
-		mcp.WithDescription("Get top operations for a specific service. Defaults to last 24 hours if no time specified."),
+		mcp.WithDescription("Get top operations for a specific service. Defaults to last 5 hours if no time specified."),
```

Comment:
```suggestion
		mcp.WithDescription("Get top operations for a specific service. Defaults to last 6 hours if no time specified."),
```

Can we do 6 hours instead of 5? The reason is consistency with the product. The defaults we have in the product is 30m, 1h, 3h, 6h